spec/controllers/idv/in_person_controller_spec.rb

```diff
@@ -82,7 +82,7 @@
               it 'redirects to the first step' do
                 get :index
 
-                expect(response).to redirect_to idv_in_person_proofing_state_id_path
+                expect(response).to redirect_to idv_in_person_proofing_state_id_url
```

Comment:
Maybe change all to `idv_in_person_proofing_state_id_path` in this file- we talked about this